src/tools/debugger/entities/SteppingType.java

```diff
@@ -104,11 +110,37 @@ public void process(final Suspension susp) {
     }
   },
 
+  @SerializedName("stepToMessageRcvr")
+  STEP_TO_RECEIVER_MESSAGE("stepToMessageRcvr", "Step to Msg Receiver", Group.ACTOR_STEPPING, "msg-open", new Class[] {EventualMessageSend.class}, new ActivityType[] {ActivityType.ACTOR}) {
```

Comment:
With this approach, you can't step on messages that are send by a CSP process, a fork/join task, or a `Thread`.
And, I don't see the benefit of restricting it to actors.